w09h01/test/pgdp/pingutrip/TestreadWayPoints.java

```diff
@@ -0,0 +1,83 @@
+package pgdp.pingutrip;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static pgdp.pingutrip.PinguTrip.readWayPoints;
+
+public class TestreadWayPoints {
+    @Test
+    @DisplayName("No end (---)")
+    void test1() {
```

Comment:
It would be great to use method names that describing the use of the test more, like:

```suggestion
    void testWithNoEnd() {
```